rebuild/config/_breakpoints.scss

```diff
@@ -7,3 +8,6 @@ $tablet-landscape-upper: 1199px;
 $desktop-lower: 1200px;
 $desktop-upper: 1799px;
 $big-desktop-lower: 1800px;
+
+//Max width for pages layouts
+$page-max-width: $big-desktop-lower;
```

Comment:
Agree RE this file not being the right place for this – I'd either have it directly in the config file like we do for the assets dir, or create a new file for layout config items, e.g. this and the L/R padding